docs/docs/getting-started/installation.md

```diff
@@ -1,51 +1,108 @@
 
-After installing Python, it's recommended that you set up a virtual environment. This will isolate your Dagster project from the rest of your system and make it easier to manage dependencies.
+### Using `create-dagster`
 
-There are many ways to do this, but this guide will use `venv` as it doesn't require additional dependencies.
+The `create-dagster` utility can be installed using [Homebrew](https://brew.sh/), `curl` or invoked without installation using `uvx`.
 
 <Tabs>
-  <TabItem value="macos" label="MacOS">
-    ```bash
-    python -m venv venv
-    source venv/bin/activate
-    ```
-  </TabItem>
-  <TabItem value="windows" label="Windows">
-    ```bash
-    python -m venv venv
-    venv\Scripts\activate
-    ```
-  </TabItem>
+<TabItem value="uvx" label="uvx (Recommended)">
+
+First, install the Python package manager [`uv`](https://docs.astral.sh/uv/) if you don't have it:
+
+<InstallUv />
+
+Then run the `create-dagster` command using `uvx`:
+
+```bash
+uvx create-dagster project my-project
+```
+
+</TabItem>
+
+<TabItem value="brew" label="Homebrew">
+
+`create-dagster` is available in a Homebrew tap:
+
+```bash
+brew install dagster-io/tap/create-dagster
+```
+
+Then run the `create-dagster` command:
+
+```bash
+create-dagster project my-project
+```
+
+</TabItem>
+
+<TabItem value="curl" label="curl">
+
+Use `curl` to download a standalone installation script and execute it with `sh`:
+
+```bash
+curl -LsSf https://dg.dagster.io/create-dagster/install.sh | sh
+```
+
+Then run the `create-dagster` command:
+
+```bash
+create-dagster project my-project
+```
+
+</TabItem>
+
 </Tabs>
 
-:::tip
-**Looking for something more powerful than `venv`?** Try `pyenv` or `pyenv-virtualenv`, which can help you manage multiple versions of Python on a single machine. Learn more in the [pyenv GitHub repository](https://github.com/pyenv/pyenv).
-:::
+### Verifying your new project
+
+After creating your project, navigate to the project directory, activate the Python virtual environment, and verify the installation:
 
-## Installing Dagster
+```bash
+cd my-project
+source .venv/bin/activate
+dg --version
+```
 
-To install Dagster in your virtual environment, open your terminal and run the following command:
+## Alternative: Manual installation in a virtual environment
+
+If you prefer to set up Dagster manually or are installing it into an existing project, you can install Dagster in a virtual environment.
+
+### Installing Dagster
+
+<Tabs>
+<TabItem value="uv" label="uv">
+
+```bash
+uv add dagster dagster-webserver dagster-dg-cli
+```
+
+</TabItem>
+
+<TabItem value="pip" label="pip">
 
 ```bash
-pip install dagster dagster-webserver
+pip install dagster dagster-webserver dagster-dg-cli
 ```
 
-This command will install the core Dagster library and the webserver, which is used to serve the Dagster UI.
+</TabItem>
+</Tabs>
 
-## Verifying installation
+### Verifying installation
```

Comment:
I would consolidate the verifying installation section. Right now it has two code blocks that are more or less the same. I think it would be good to just combine everything so:

To verify that Dagster is installed correctly, run the following command:

```bash
> dagster --version
dagster, version 1.10.18
```

nit: I would also update the dagster version returned to 1.10.18 or something newer (it would be cool if this could be dynamic)

Reply:
Good call, consolidated and made it os-specific.

Reply:
For now I've also removed the version number itself and used `CliInvocationExample`.